lighthouse-core/test/fraggle-rock/gather/navigation-runner-test.js

```diff
@@ -114,21 +114,13 @@ describe('NavigationRunner', () => {
 
     it('should collect base artifacts', async () => {
       const {baseArtifacts} = await runner._setup({driver, config, requestedUrl});
-      expect(baseArtifacts).toMatchObject({HostUserAgent: 'Chrome', URL: {requestedUrl}});
+      expect(baseArtifacts).toMatchObject({URL: {requestedUrl}});
     });
 
     it('should prepare the target for navigation', async () => {
       await runner._setup({driver, config, requestedUrl});
       expect(mocks.prepareMock.prepareTargetForNavigationMode).toHaveBeenCalledTimes(1);
     });
-
-    it('should prepare the target for navigation *after* base artifact collection', async () => {
-      mockDriver._session.sendCommand.mockReset();
-      mockDriver._session.sendCommand.mockRejectedValue(new Error('Not available'));
-      const setupPromise = runner._setup({driver, config, requestedUrl});
-      await expect(setupPromise).rejects.toThrowError(/Not available/);
-      expect(mocks.prepareMock.prepareTargetForNavigationMode).not.toHaveBeenCalled();
-    });
```

Comment:
`sendCommand` is not called in `collectBaseArtifacts` anymore. We could have found another way to assert this, but it felt unnecessary to assert the execution order of two lines of code.

Reply:
we'll need to bring this back if we keep benchmarkindex in base artifacts, the assertion is important to ensure our benchmarkindex isn't just measuring our throttling :)